lang/fr-fr.js

```diff
@@ -31,7 +31,7 @@ export default {
 	"components.form-element.input.number.rangeUnderflow": "{minExclusive, select, true {Le nombre doit être supérieur à {min}.} other {Le nombre doit être supérieur ou égal à {min}.}}",
 	"components.form-element.input.text.tooShort": "{label} doit contenir au moins {minlength} caractères.",
 	"components.form-element.input.url.typeMismatch": "URL non valide",
-	"components.form-element.valueMissing": "{label} est requis.",
+	"components.form-element.valueMissing": "{label} est obligatoire.",
```

Comment:
I'm kinda questioning this change now. They changed from "required" to "obligatory". We use "requis" in a A LOT of terms, albeit a variety of contexts.